### DISCUSSION TITLE
new to outliners and willing to try and test

### DISCUSSION THREAD

**Discussion by Thomas-U** [vincode-io/Zavala#79] [General]
hello Maurice and all,

I just read your latest [micro.blog post](https://vincode.micro.blog/2021/04/03/keeping-busy.html) and got curious about Zavala.

I have never used an outliner before and have so far always used a combo of manual notes and note-taking apps with (sub-)paragraphs to organize my projects.

As Zavala does not come with a manual as of now I perused the web and found this article to give quite a good overview as to what outliners can do: [https://zapier.com/blog/best-outline-software/](https://zapier.com/blog/best-outline-software/) (What to Look For...)

I spent the last hour to transfer some of my current projects to Zavala. Thus I'll find out what works for me and what is missing or flawed.

I have installed Zavala on my M1 Mac mini and via Testflight on iPhone 12 Pro Max and iPad Pro 2018. iCloud sync works fast and flawless.

So: Count me in your test team. I'm looking forward to another great cooperation.

**Comment by vincode-io** [comment]
Welcome @Thomas-U! Seeing this post from you made my day.

Thanks for the article link. I hadn't seen that one before. I'll go through it and see if there are any interesting ideas that I should incorporate.
